File: URI-C++/1036.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main() {
	
	double a, b, c, delta, r1, r2;
	
	cin >> a >> b >> c;
	
	delta = pow(b,2) - (4 * a * c);
	
	r1 = (-b + sqrt(delta)) / (2 * a);
	r2 = (-b - sqrt(delta)) / (2 * a);
	
	if (r1 == 0 || r2 == 0 || delta <= 0 || a == 0) {
		cout << "impossivel calcular" << endl;
		return 0;
	}
	else {
		cout << "R1 = " << fixed << setprecision(5) << r1 << endl;
		cout << "R2 = " << fixed << setprecision(5) << r2 << endl;
	}
	
	return 0;
}

File: Java/1036.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {

        Locale.setDefault(Locale.US);
        Scanner entrada = new Scanner(System.in);

        double a = entrada.nextDouble();
        double b = entrada.nextDouble();
        double c = entrada.nextDouble();

        double delta, r1, r2;

        if(a == 0) {
            System.out.println("Impossivel calcular");
            return;
        }
        else {
            delta = Math.pow(b, 2) - (4 * a * c);

            r1 = (-b + Math.sqrt(delta)) / (2 * a);
            r2 = (-b - Math.sqrt(delta)) / (2 * a);

            if (delta < 0) {
                System.out.println("Impossivel calcular");
                return;
            }
            else {
                System.out.printf("R1 = %.5f%n", r1);
                System.out.printf("R2 = %.5f%n", r2);
            }
        }
        
        entrada.close();
    }
}